avaframe/out3Plot/plotUtilsCfg.ini

```diff
@@ -51,3 +51,4 @@ namepfd = peak flow depth
 namepfv = peak flow velocity
 namerelTh = release thickness
 namefd = flow depth
+namev = flow velocity
```

Comment:
```suggestion
namefv = flow velocity
```

Reply:
I made an issue out of this, as this has to be then renamed everywhere in com1DFAPy